cg/services/illumina_services/illumina_post_processing_service/illumina_post_processing_service.py

```diff
@@ -27,36 +30,51 @@ def __init__(self, status_db: Store, housekeeper_api: HousekeeperAPI, dry_run: b
 
```

Comment:
fix the dry run in the constructor, it doesn't use the dry run given as parameter